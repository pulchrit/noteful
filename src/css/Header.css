.header {
    padding: 20px;
    width: 100%;
    text-align: center;
    border-bottom: 1px solid darkgrey;
    font-size: 2em;
}