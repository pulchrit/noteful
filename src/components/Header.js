import React from 'react';
import {Link} from 'react-router-dom';
import '../css/Header.css';

const Header = () => (
    
        <header className='header'>
            <h1>
            <Link to='/'>Noteful</Link>
            </h1>
        </header>

);

export default Header;

