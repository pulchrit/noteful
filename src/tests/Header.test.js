import React from 'react';
import ReactDOM from 'react-dom';
import {BrowserRouter} from 'react-router-dom';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Header from "../components/Header";

describe("Header component", () => {
    it("renders without crashing", () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>,
            div
        );
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders UI as expected', () => {
        const wrapper = shallow(<Header />);
        expect(toJson(wrapper)).toMatchSnapshot();
      });
});