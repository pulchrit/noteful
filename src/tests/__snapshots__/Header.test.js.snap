// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders UI as expected 1`] = `
<header
  className="header"
>
  <h1>
    <Link
      to="/"
    >
      Noteful
    </Link>
  </h1>
</header>
`;
